indico/modules/events/papers/templates/display/_common.html

```diff
@@ -0,0 +1,42 @@
+{% macro render_cfp_infoline(event, cfp, show_icon=true, redirect_first=false) %}
+    {% set can_submit_papers = cfp.is_open or session.user %}
```

Comment:
This check makes no sense - being logged in is not enough to submit a paper. Since we don't have a user-specific deadline override like we do in abstracts the only thing to check here is whether CFP is open